processor/file_test.go

```diff
@@ -227,8 +234,8 @@ func TestNewFileJobFullname(t *testing.T) {
 func TestNewFileJob(t *testing.T) {
 	ProcessConstants()
 
-	fi, _ := os.Stat("./examples/issue114/java")
-	job := newFileJob("./examples/issue114/", "java", fi)
+	fi, _ := os.Stat("../examples/issue114/java")
+	job := newFileJob("../examples/issue114/", "java", fi)
```

Comment:
I'm not sure why I had to change these. Has something about the testing changed?

Reply:
No... that's odd... 

How were you running them?

Reply:
Both `go test ./...` and `sh test-all.sh` - the same way I've run the tests in the past.